### ISSUE TITLE
"Save as... latex" bug with ObjectProperties with inverseOf relations 

### ISSUE THREAD

**Issue by jorisslob** [protegeproject/protege#94]
The following minimal example ontology:

```
@prefix : <http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61#> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@base <http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61> .

<http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61> rdf:type owl:Ontology .

:buzz_of rdf:type owl:ObjectProperty ;         
         owl:inverseOf :fizzle_of .

:fizzle_of rdf:type owl:ObjectProperty .
```

generates the following Latex when saved as latex:

```
\documentclass{article}
\parskip 0pt
\parindent 0pt
\oddsidemargin 0cm
\textwidth 19cm
\begin{document}

\section*{Object properties}\subsubsection*{buzz\_of}

<http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61#buzz_of>~\ensuremath{\equiv}~<http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61#fizzle_of>\ensuremath{^-}

\subsubsection*{fizzle\_of}

<http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61#buzz_of>~\ensuremath{\equiv}~<http://www.semanticweb.org/jslob/ontologies/2014/9/untitled-ontology-61#fizzle_of>\ensuremath{^-}

\section*{Data properties}\section*{Individuals}\section*{Datatypes}\end{document}
```

Note that in the inverse relations, the entire base url is displayed, which is unwanted, but also special characters are not escaped (the underscore and brackets). This causes the resulting tex file to be invalid.

This is an seems to be an owlapi problem. Maybe fixed when going to the latest version? I haven't tested this.

**Comment by jgrzebyta** [comment]
I still have this error.

protege: version 5.0 Build beta-23

**Comment by matthewhorridge** [comment]
This is an OWL API problem.  I'll report it on that tracker.

**Comment by ignazio1977** [comment]
Assuming that this is the expected output:

    \documentclass{article}
    \parskip 0pt
    \parindent 0pt
    \oddsidemargin 0cm
    \textwidth 19cm
    \begin{document}
    \section*{Object properties}\subsubsection*{buzz\_of}
    buzz\_of~\ensuremath{\equiv}~fizzle\_of\ensuremath{^-}
    \subsubsection*{fizzle\_of}
    buzz\_of~\ensuremath{\equiv}~fizzle\_of\ensuremath{^-}
    \section*{Data properties}\section*{Individuals}\section*{Datatypes}\end{document}

The bug is now fixed.

**Comment by matthewhorridge** [comment]
Needs double checking in the latest 5.5.0-beta

**Comment by matthewhorridge** [comment]
Works as expected (fixed with the upgraded OWL API) in 5.5.0-beta-8
